### PR TITLE
Feature/10347 deploy new development env

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#10347](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10347)
Story: [AB#7746](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7746)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -595,102 +559,40 @@ jobs:
...
-      displayName: 'Terraform Destroy (Key Vault)'
-      inputs:
-        provider: 'azurerm'
-        command: 'destroy'
-        workingDirectory: '$(build.artifactStagingDirectory)/generateKeyVault'
-        commandOptions: '-var="environment=$(release)" -var="keyvault=gpit-$(release)-kv" -var="keyvaultrg=$(release)-kv-rg" -var="kv_addrprefix=10.201" -var="kv_sqlusername=gpitfbcadmin" -var="kv_coreurl=$(release).dynamic.buyingcatalogue.digital.nhs.uk" -var="region=uksouth" -var="subscription_id=7b12a8a2-f06f-456f-b6f9-aa2d92e0b2ec" -var="tenant_id=50f6071f-bbfe-401a-8803-673748e629e2" -var="pjtcode=gpit" -var="project=gpitfutures"'
-        environmentServiceNameAzureRM: 'NHSAPP-BuyingCatalogue (Non-Prod)'
+  - publish: $(build.artifactStagingDirectory)
+    artifact: build-artifact 
+
```
Comment (PhysicsJon) [inline azure-pipelines.yml]: missing newline

File: generateKeyVault/keyvault_secrets.tf [hunk 2]
```diff
@@ -63,3 +63,54 @@ resource "azurerm_key_vault_secret" "kv_coreurl" {
...
+  key_vault_id = azurerm_key_vault.keyvault.id
+  
+  tags = {
+    environment = var.environment
+  }
+
+  depends_on = [
+    azurerm_management_lock.keyvault_lock
+  ]
+}
```
Comment (PhysicsJon) [inline generateKeyVault/keyvault_secrets.tf]: missing newline

File: terraform/z_imported_secrets.tf [hunk 3]
```diff
@@ -17,3 +17,8 @@ data "azurerm_key_vault_secret" "coreurl" {
   name         = "${var.pjtcode}${local.shortenv}coreurl"
   key_vault_id = local.kv_id
 }
+
+data "azurerm_key_vault_secret" "certname" {
+  name         = "${var.pjtcode}${local.shortenv}certname"
+  key_vault_id = local.kv_id
+}
```
Comment (PhysicsJon) [inline terraform/z_imported_secrets.tf]: missing newline

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
All good, except for the missing newlines

File: azure-pipelines.yml (hunk 1 above)
Comment (gavinwakeley1) [reply azure-pipelines.yml]: Done

File: generateKeyVault/keyvault_secrets.tf (hunk 2 above)
Comment (gavinwakeley1) [reply generateKeyVault/keyvault_secrets.tf]: Done

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**